relays/ethereum/src/substrate_client.rs

```diff
@@ -317,7 +317,8 @@ impl SubmitEthereumExchangeTransactionProof for SubstrateRpcClient {
 		let nonce = self.next_account_index(account_id).await?;
 
 		let transaction = create_signed_transaction(
-			bridge_node_runtime::Call::BridgeCurrencyExchange(
+			// TODO: Change so that that it's dynamic
```

Comment:
Can you add a specific issue number? I believe it's going to be #209

Reply:
Yeah, it's related. I can add it